worknenjoy/gitpay
# Account settings

## Description

> Add Account Settings page

## Changes

- Account Settings 

## Issue

> Closes #535 

## Changes

> ``` /profile/account-settings ```

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

- Create & log in
- Visit url ``` /profile/account-settings```
- Use provider buttons for Github?Bitbucket
- Or for updating local account, fill in the details and check the privacy policy checkbox and click the update account button
- For updating any fields of the local account, the old password and the privacy-policy checkbox are mandatory
- The new password field and confirm password field must have the same value

## Before
> Screenshot from the state before
![image](https://user-images.githubusercontent.com/45919359/83884630-c6132b00-a762-11ea-983a-009cf7dae72f.png)


## After
> Screenshot from the state after your Pull Request
![image](https://user-images.githubusercontent.com/45919359/83884468-8ba98e00-a762-11ea-87f7-293fbabd784a.png)
![image](https://user-images.githubusercontent.com/45919359/83884591-b693e200-a762-11ea-8130-2fb5a97ff5e6.png)
